01week/datatypes.js

```diff
@@ -72,11 +73,3 @@ function trueFunctionII() {
     "1 of 2 statements is true : " + text;
 }
 console.log(trueFunctionII());
-

```

Comment:
I need your function to evaluate to true if one parameter evaluates to true. need to use parameters.